code/datums/gamemode/role/ninja.dm

```diff
@@ -523,6 +553,91 @@ Helpers For Both Variants
 	else
 		return ..()
 
+		
+/*=======
+Suit and assorted
+=======*/
+
+/obj/item/clothing/head/helmet/space/ninja
+	name = "elite ninja hood"
+	desc = "What may appear to be a simple black garment is in fact a highly sophisticated nano-weave helmet. Standard issue ninja gear."
+	icon_state = "s-ninja"
+	item_state = "s-ninja"
+	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 25)
+	species_fit = list("Human")
+	species_restricted = list("Human")
+	eyeprot = 3
+	body_parts_covered = FULL_HEAD|BEARD
+	
+/obj/item/clothing/head/helmet/space/ninja/apprentice
+	name = "ninja hood"
+	desc = "What may appear to be a simple black garment is in fact a sophisticated nano-weave helmet. Standard issue ninja apprentice gear."
+	eyeprot = 0
+	pressure_resistance = ONE_ATMOSPHERE
+	armor = list(melee = 50, bullet = 15, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
+	
+/obj/item/clothing/head/helmet/space/ninja/apprentice/New()
```

Comment:
needs periodic warning messages I think to prevent too many people from getting dumped in space

the problem with just adding a message at the start is that it will probably be buried in the wall of ninja spawned text, internals active message, etc.

Reply:
They currently get `<span class='danger'>You are currently on a direct course to the station. Find a way inside before your suit's life support systems give out.` which is going to be the last thing on their chatbar as soon as they spawn.
And since they won't be doing much but fly for the first couple of seconds, I doubt they'll miss it.

Reply:
Not sure where my comment went, but.
They get a message saying 'get the fuck inside' in the greet which is the last thing they get in the text when they spawn as ninja and since they're busy doing nothing because they're being flung it's likely that they'll read it anyway.